frame/staking/src/pallet/mod.rs

```diff
@@ -210,6 +210,12 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxNominatorRewardedPerValidator: Get<u32>;
 
+		#[pallet::constant]
+		type ExposurePageSize: Get<u32>;
+
+		#[pallet::constant]
+		type ExposureMaxPages: Get<u32>;
```

Comment:
I would consider skipping this. Later on, once we do e2e paginated election, we would introduce some bound on it, but for now it is reasonable to assume it is unbounded.